sql/moz-fx-data-shared-prod/telemetry/releases_latest/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Releases Latest
+description: |-
+  This is built on top of telemtry.releases and aims
+  to provide an entry on each day for release and beta channels
```

Comment:
It appears that you're also handling ESR (which is good).  Can you put the values in double quotes so that it's clear that these are the literal values?  I.e.,
```
on each day for the major release channels ("release", "beta", "esr")
```